tests/ut/test_get_vector.cc

```diff
@@ -17,25 +17,81 @@
 #include "knowhere/factory.h"
 #include "utils.h"
 
-TEST_CASE("Test Get Vector By Ids", "[GetVectorByIds]") {
+TEST_CASE("Test Binary Get Vector By Ids", "[Binary GetVectorByIds]") {
     using Catch::Approx;
 
     int64_t nb = 10000;
     int64_t dim = 128;
     int64_t seed = 42;
 
-    auto base_gen = [&]() {
+    auto base_bin_gen = [&]() {
         knowhere::Json json;
         json[knowhere::meta::DIM] = dim;
-        json[knowhere::meta::METRIC_TYPE] = knowhere::metric::L2;
+        json[knowhere::meta::METRIC_TYPE] = knowhere::metric::HAMMING;
```

Comment:
how about other binary metrics?

Reply:
for binary index types, metric type does not matter